src/slack/slack.js

```diff
@@ -630,6 +636,7 @@ export class Slack {
           office: order.office,
           reason: order.reason,
           isUrgent: order.isUrgent,
+          manager: order.manager,
           ...this.variant === 'wincent' ? {} : {spinoff: order.spinoff},
```

Comment:
also here special handling needed:
```suggestion
          ...this.variant === 'wincent' ? {} : {spinoff: order.spinoff, manager: order.manager},
```

Reply:
fixed